public/assets/js/main.js

```diff
@@ -1,12 +1,12 @@
-async function getLakes () {
-  const res = await fetch('/api/lake/all')
-  const lakes = await res.json()
-  lakes.forEach(lake => createCard(lake))
+async function getLakes() {
+  const res = await fetch("/api/lake/all");
+  const lakes = await res.json();
+  lakes.forEach((lake) => createCard(lake));
 }
 
-function createCard (lake) {
-  const card = document.createElement('div')
-  card.classList.add('card')
+function createCard(lake) {
+  const card = document.createElement("div");
+  card.classList.add("card");
```

Comment:
only changes here are that you added semicolons, I presume your formatter (e.g. prettier) did that. Can you remove them everywhere?